### ISSUE TITLE
Fetch value from a MyView inherited from AbstractView

### ISSUE THREAD

**Issue by nadjibnet** [HomeSeer/Plugin-SDK#125]
**Environment and System Config**
- OS: Windows
- Language: C#
- IDE: VS2019
- HomeSeer Version: v4.1.8
- PluginSDK Version: v1.2.1

**Describe the issue**
I need help to create my custom view to be used on event trigger, this view contains lines, each line contains inputs + removeButton. So the user can add a line and remove.

I was able to create the UI for this trigger flowing the source code of InputView, ButtonView... But, I didn't find a way to fetch the user inputted data of those inputs (html component).

I tired to add properties for those inputs but I did know what to put on [JsonProperty("stringValue1")]
the button is performing the remove action on js and using jQuery. (So, no problem on this part.)

Please, do you have any suggestion to get user inputted values from my view ?

**Comment by nadjibnet** [comment]
I solved my issue by creating a custom html inside the trigger and post data using the same logic that the plugin use

**Comment by spudwebb** [comment]
Can you elaborate on what you mean with "creating custom html inside the trigger" ? How did you do this?

**Comment by nadjibnet** [comment]
> Can you elaborate on what you mean with "creating custom html inside the trigger" ?

An advanced trigger will contains more that a SelectList, Checkbox, TextBoxes that are present on the Plugin-SDK.

Some missing view, like buttons, a list (dynamic) that the user can add more items on it, and more...

> How did you do this?
I created a class inherited from AbstractView, and I load my html&js from a html_file.
I made a communication between the HS4 and my html part to save data.
